docker/docker-compose.dev.yml

```diff
@@ -6,16 +6,12 @@ services:
     volumes:
       - ../docker/provision:/home/provision
       - ../server:/home/viame_girder
-      - ./data/assetstore:/home/assetstore
-      - pipelines:${VIAME_PIPELINES_PATH:-/home/VIAME/pipelines}:ro
-      - type: bind
-        source: ${PUBLIC_DATA_PATH}
-        target: /data/public
-        read_only: true
-        volume:
-          nocopy: true
     entrypoint: /home/provision/girder_entrypoint_dev.sh
 
+  girder_worker:
+    volumes:
+      - ../server:/home/viame_girder
+
```

Comment:
Is this to cover changes to the girder worker tasks?

Reply:
once girder worker issues get resolved, you will be able to run local changes with `ldc dev girder_worker`.

Reply:
Yeah, you'll be able to `ldc dev girder_worker`